WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js

```diff
@@ -357,56 +357,90 @@ function() {
 	return appCtxt.getById(this.folderId);
 };
 
-ZmBriefcaseBaseItem.prototype._loadFromDom =
-function(node) {
-
-	this.id = node.id;
-
-	if (node.rest)	{ this.restUrl = node.rest; }
-	if (node.l)		{ this.folderId = node.l; }
-	if (node.name)	{ this.name = node.name; }
-	if (node.cr)	{ this.creator = node.cr; }
-	if (node.perm)	{ this.permission = node.perm; }
+ZmBriefcaseBaseItem.prototype._loadFromDom = function (node) {
+    /**
+     * In search api response we get 'id' and 'sfid' for files in 'files shared with me' folder. Here id is a combination id which is required to form the rest url
+     * and get the prview data and sfid is the actual 'id' of the item/file.
+     * To delete the file need to call "ItemActionRequest' with 'sfid' coming from 'searchResponse' since this is the actual file id.
+     *  Interchanged the 'id' and 'sfid' properties for items in 'files shared with me' folder, so that rest of the operations is perfomed as earlier.
+     */
+    if (node.sfid && node.l == ZmFolder.ID_FILE_SHARED_WITH_ME) {
```

Comment:
looks like indentation is done using spaces, in same file it's done using tabs

Reply:
As per the standard I have done using spaces. But earlier indentation  are in tabs.  Let me know which one should we keep.

Reply:
converted it to tabs.